litho-widget/src/main/java/com/facebook/litho/widget/HorizontalScrollSpec.java

```diff
@@ -203,6 +205,14 @@ public boolean onPreDraw() {
         new ViewTreeObserver.OnScrollChangedListener() {
           @Override
           public void onScrollChanged() {
+            if (onScrollChangeListener != null) {
+              onScrollChangeListener.onScrollChange(
```

Comment:
Our lint caught that `View.OnScrollChangeListener` is available only from API23 while we support API15+. Can you find alternative way of achieving this?

Reply:
..or API-version-guard it 😄 !

Reply:
VerticalScrollSpec uses NestedScrollView.OnScrollChangeListener in the same way, and it's API24+. I don't see how it's being guarded there.

In my PR, we're only using the interface, so as long as Litho is built with SDK 23+, it should work. I'm not calling any SDK methods that needed to be guarded against.

Reply:
Actually `NestedScrollView` is implemented in support library and they provide their own OnScrollChangedListener, so that one is safe to use: 

https://android.googlesource.com/platform/frameworks/support/+/refs/heads/support-library-27.1.1/core-ui/src/main/java/android/support/v4/widget/NestedScrollView.java#84

Reply:
You could create that interface inside spec instead